Sources/Common/Extensions/StringExtensions.swift

```diff
@@ -31,6 +31,11 @@ public extension String {
         String((0 ..< length).map { _ in abcLetters.randomElement()! })
     }
 
+    static func randomStringWithSpecialCharacters(length: Int = 20) -> String {
```

Comment:
If we do end up using this function, move it into `Tests/Shared` since it's a function used only for tests.